draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -425,6 +434,11 @@ const TitleBlockZen = ({
   const [isSmallOrMediumViewport, setSmallOrMediumViewport] = React.useState(
     false
   )
+  const hasNavigationTabs = navigationTabs && navigationTabs.length > 0
+  const collapseNavigationArea =
+    collapseNavigationAreaWhenPossible &&
+    (navigationTabs === undefined || navigationTabs) &&
```

Comment:
Having a bit of trouble understanding the intention of the right side of the `||` here.

"If `navigationTabs` is undefined or `navigationTabs` is truthy"?

Reply:
Oh, that is because I am dumb and didn’t rewrite it when I extracted the intermediate var.

Reply:
Should be fixed now, @sebpearce